.github/workflows/build.yml

```
@@ -0,0 +1,48 @@
+name: Build and Test
+
+on:
+    push:
+      branches: [ "main" ]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)